src/app/features/report/summary/test-summary-table.component.html

```diff
@@ -14,13 +14,13 @@ <h2 class="mb-2 mt-4">Test Summary by {{categoryName}}</h2>
 				{{element.name}}
 			</div>
 			<div class="col-1 cell">
-				{{element.getPassedPercentage()}}
+				{{element.passed}} - {{element.getPassedPercentage()}}
```

Comment:
Just a suggestion. I reckon that the "desired" format (when the issue was submitted back in 2018) was this is one, but:
* would not it be more readable with parenthesis?: "6 (67%)" "3 (33%)" "0 (0%)"?
* it would become even more readable if the quantities were in bold

Reply:
Or an alternate layout would be 2 rows for each table:
 * row 1 with the quantities
 * row 2 with the percentages

Reply:
option 1:
![image](https://github.com/user-attachments/assets/552ec809-552b-4d57-ac6d-590efabeaf75)

Reply:
option 2:
![image](https://github.com/user-attachments/assets/629e7c55-a821-4f1d-82bf-766299f760ca)

Reply:
@Franek75 , which one is the most preferable in your opinion?

Reply:
timeout here! uploading the second option...